pylatex/basic.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+This module implements several classes that represent basic latex commands.
+
+..  :copyright: (c) 2014 by Jelte Fennema.
+    :license: MIT, see License for more details.
+"""
+
+from .base_classes import CommandBase, Environment, ContainerCommand
+from .package import Package
+
+
+class BasicCommand(CommandBase):
```

Comment:
Can you also do the same here? The extra class is not really needed. It is fine to just do something special for NewPage and then inherit the others from NewPage.